.vscode/launch.json

```diff
@@ -9,7 +9,8 @@
             "args": ["--extensionDevelopmentPath=${workspaceFolder}" ],
             "sourceMaps": true,
             "outFiles": [ "${workspaceFolder}/out/**/*.js" ],
-            "preLaunchTask": "npm: watch"
+            "preLaunchTask": "build",
```

Comment:
Miscellaneous: make the launch task also build the pkl-lsp CLI (see changes in .vscode/tasks.json)